src/content/docs/infrastructure/host-integrations/host-integrations-list/port-monitoring-integration.mdx

```diff
@@ -13,8 +13,90 @@ freshnessValidatedDate: never
 
 Our port monitoring integration monitors status for networking ports, such as TCP, UDP, etc., and reports data to New Relic.
 
-To get started, see the [Port monitoring on-host integration README](https://github.com/newrelic/nri-port-monitor) on GitHub.
-
 <Callout variant="tip">
   This integration is released as [open source on Github](https://github.com/newrelic/nri-port-monitor). A change log is also available there for the latest updates. Support for this integration is available with an [Expert Services subscription](https://www.newrelic.com/expertservices).
 </Callout>
+
+## Install the integration 
+
+Follow the steps below to install the integration:
+
+<Steps>
+    <Step>
+## Install the infrastructure agent [#infra-install]
+
+To use the Port Monitoring integration, you need to first [install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic/) on the same host. All our on-host integrations require the infrastructure agent, which helps expose and report metrics to New Relic.  
+
+    </Step>
+    <Step>
+## Download and prepare the integration [#download]
+
+1. Download and unpack the ZIP file from [the GitHub repo](https://github.com/newrelic/nri-port-monitor/releases):
+
+    ```shell
+    wget https://github.com/newrelic/nri-port-monitor/releases/download/1.3/nri-port-monitor.tar.gz
+    ```
+
+    ```shell
+    tar -zxvf nri-port-monitor.tar.gz 
+    ```
+
+2. Copy the `bin` directory with `nri-port-monitor` executable to `/var/db/newrelic-infra/newrelic-integrations`. Do the same for the `port-monitor-definition.yml` config file:
+
+    ```shell
+    cp nri-port-monitor/bin/port-monitor /var/db/newrelic-infra/newrelic-integrations/bin/
+    cp nri-port-monitor/port-monitor-definition.yml /var/db/newrelic-infra/newrelic-integrations/
+    ```
+
+3. Set execution permissions for the binary file `nr-port-monitor`:  
```

Comment:
What if someone doesn't want to add execution permissions to this binary file? Or doesn't have admin permissions on their server? What then? Is it worth providing some context about why we want them to do this?